src/main/java/seedu/address/model/person/Birthday.java

```diff
@@ -0,0 +1,63 @@
+package seedu.address.model.person;
+
+import seedu.address.commons.exceptions.IllegalValueException;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+/**
+ * Represents a Person's address in the address book.
+ * Guarantees: immutable; is valid as declared in {@link #isValidBirthday(String)}
+ */
+public class Birthday {
+    public static final String MESSAGE_BIRTHDAY_CONSTRAINTS =
+            "Person Birthday should be in the format of DD/MM/YYYY";
+
+    public final String value;
+
+    /**
+     * Validates given birthday.
+     *
+     * @throws IllegalValueException if given address string is invalid.
+     */
+    public Birthday(String birthday) throws IllegalValueException {
+        if (!isValidBirthday(birthday)) {
+            throw new IllegalValueException(MESSAGE_BIRTHDAY_CONSTRAINTS);
+        }
+        this.value = birthday;
+    }
+
+    /**
+     * Returns true if a given string is a valid person email.
+     */
+    static boolean isValidBirthday(String birthday) {
+
+        SimpleDateFormat format = new SimpleDateFormat("dd/MM/yyyy");
+        format.setLenient(false);
```

Comment:
only this format for now. checks for invalid dates e.g. 30 feb

Reply:
This format is good enough. It is widely used in Asia region I think.